src/dal-plugin/Stream.mm

```diff
@@ -127,7 +127,7 @@ - (NSSize)testCardImageSize {
 
 - (NSImage *)testCardImage {
     if (_testCardImage == nil) {
-        _testCardImage = ImageOfTestCardWithSize([self testCardImageSize]);
+        _testCardImage = ImageOfTestCardWithSize([self testCardSize]);
```

Comment:
Likewise you could also consider making this `_testCardImage = ImageOfTestCardWithSize(self.testCardSize);`